server/routes/api/auth.js

```diff
@@ -1,25 +1,26 @@
-const express = require('express');
-const router = express.Router();
-const bcrypt = require('bcrypt');
+const express = require('express')
+const router = express.Router()
+const bcrypt = require('bcrypt')
 
 // User Model
-const User = require('../../models/user');
+const User = require('../../models/user')
 
 // POST User Login
 router.post('/login', (req, res) => {
+  //doesn't User.authenticate need to be returned here?
```

Comment:
If you have a question like this, it's usually better to just ask in Slack. We want to minimize comments like this. If there's a task to do, we should create a ticket around it. Otherwise, these types of things go unnoticed, forgotten, and become stale.

It looks like the rest of the changes in this file are just linter adjustments. Let's just revert this file completely and remove it from the diff.